### PR TITLE
spice_js_api.md - Adding documentation & Examples

### PR DESCRIPTION
**Pull request by GregorioMartinez**
Added example for strip_html
Added documentation and examples for isNumber, scaleToFit, and strip_href

### CODE REVIEW

File: duckduckhack/spice/spice_js_api.md [hunk 1]
```diff
@@ -142,6 +142,25 @@ Removes HTML tags/characters from a string
...
+- `strip_html("<a href='http://duckduckgo.com'>DuckDuckGo</a>'")` -> `"DuckDuckGo"`
+
+## strip_href(html)
+Removes src and href attributes from a string
+
+Note: The src and href attributes are not removed entirely. They are prefixed with _ 
+
+**Parameters**
+
+**html** : *string*, String containing href or src attribute
```
Comment (moollaza) [inline duckduckhack/spice/spice_js_api.md]: let's wrap `href` and `src` in backticks so they display as code.

File: duckduckhack/spice/spice_js_api.md [hunk 2]
```diff
@@ -142,6 +142,25 @@ Removes HTML tags/characters from a string
...
 
+Example:
+	
+- `strip_html("<strong>DuckDuckGo</strong>")` -> `"DuckDuckGo"`
+- `strip_html("<a href='http://duckduckgo.com'>DuckDuckGo</a>'")` -> `"DuckDuckGo"`
+
+## strip_href(html)
+Removes src and href attributes from a string
+
+Note: The src and href attributes are not removed entirely. They are prefixed with _
```
Comment (moollaza) [inline duckduckhack/spice/spice_js_api.md]: let's wrap the underscore in backticks or quotes: `_` or "_"

File: duckduckhack/spice/spice_js_api.md [hunk 3]
```diff
@@ -272,6 +305,25 @@ Example:
  - `toHTTP("https://duckduckgo.com")` -> `"http://duckduckgo.com"`
  - `toHTTP("duckduckgo.com")` -> `"duckduckgo.com"`
 
+
+## scaleToFit(width, height, maxWidth, maxHeight)
+Returns an object with a with width and height properties scaled proportionally to fit within a specified range
```
Comment (moollaza) [inline duckduckhack/spice/spice_js_api.md]: whoops, looks like a typo: "with a with width"

**Comment by moollaza** [comment]
@GregorioMartinez as always, great job, thanks a ton for taking the time to do this!

**Comment by GregorioMartinez** [comment]
@moollaza Fixed typo, some grammar, and formatting. Thanks for catching that.

**Comment by moollaza** [comment]
@GregorioMartinez nice! Thanks again. This LGTM :+1:

**Pull request merged**